supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-enterprise-linux.adoc

```diff
@@ -67,3 +67,834 @@ For documentation questions, contact rhel-docs@redhat.com.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:kerberos[kerberos]
+
+[discrete]
+[[kerberos-realm]]
+==== image:images/yes.png[yes] Kerberos realm (noun)
+*Description*: A Kerberos realm encompasses all the principals managed by a Kerberos Key Distribution Center (KDC). In an IdM deployment, the Kerberos realm includes all IdM users, hosts, and services.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:idm-deployment[idm-deployment], xref:key-distribution-center[key-distribution-center]
+
+[discrete]
+[[kerberos-ticket-policies]]
+==== image:images/yes.png[yes] Kerberos ticket policies (noun)
+*Description*: The Kerberos Key Distribution Center (KDC) enforces ticket access control through connection policies, and manages the duration of Kerberos tickets through ticket lifecycle policies. For example, the default global ticket lifetime is one day, and the default global maximum renewal age is one week.
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/{ProductNumberLink}/html/configuring_and_managing_identity_management/managing-kerberos-ticket-policies_configuring-and-managing-idm#idm-kerberos-ticket-policy-types_managing-kerberos-ticket-policies[IdM Kerberos ticket policy types].
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:key-distribution-center[key-distribution-center]
+
+[discrete]
+[[key-distribution-center]]
+==== image:images/yes.png[yes] Key Distribution Center (KDC) (noun)
+*Description*: The Kerberos Key Distribution Center (KDC) is a service that acts as the central, trusted authority that manages Kerberos credential information. The KDC issues Kerberos tickets and ensures the authenticity of data originating from entities within the IdM network.
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/{ProductNumberLink}/html/configuring_and_managing_identity_management/managing-kerberos-ticket-policies_configuring-and-managing-idm#role-of-idm-kdc_managing-kerberos-ticket-policies[The role of the IdM KDC].
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[ldap]]
+==== image:images/yes.png[yes] LDAP (noun)
+*Description*: The Lightweight Directory Access Protocol (LDAP) is an open, vendor-neutral,  application protocol for accessing and maintaining distributed directory information services over a network. Part of this specification is a directory information tree (DIT), which represents data in a hierarchical tree-like structure consisting of the Distinguished Names (DNs) of directory service entries. LDAP is a "lightweight" version of the Directory Access Protocol (DAP) described by the ISO X.500 standard for directory services in a network.
```

Comment:
Extra space in "n open, vendor-neutral,  application protocol for "

Reply:
+1